taskcluster/scripts/fetch/generate_vsstudio_bundle.ps1

```diff
@@ -35,7 +35,7 @@ if(!(Test-Path "C:\VisualStudio\VC\Auxiliary\Build\vcvars64.bat")){
         --add Microsoft.VisualStudio.Component.VC.Redist.MSM `
         --add Microsoft.VisualStudio.Component.VC.CoreBuildTools `
         --add Microsoft.VisualStudio.Component.VC.CoreIde `
-        --add Microsoft.VisualStudio.Component.Windows10SDK.19041 `
+        --add Microsoft.VisualStudio.Component.Windows10SDK.20348 `
```

Comment:
This would require the vsstudio bundle to be updated aswell. it's currently not generated in ci.